src/main/java/com/hoodiecoder/enchantmentcore/CustomEnch.java

```diff
@@ -312,19 +290,10 @@ public String getName() {
      * @return The starting level of the enchantment
      */
     @Override
-    public final int getStartLevel() {
```

Comment:
This method was supposed to be final so it wasn't possible to make the start level anything different (all enchantments start at level 0). This should be changed back, or tell me why you changed this